core/src/main/java/io/grpc/util/OutlierDetectionLoadBalancer.java

```diff
@@ -814,6 +854,8 @@ public void ejectOutliers(AddressTrackerMap trackerMap, long ejectionTimeNanos)
         // If the failure rate is above the threshold, we should eject...
         double maxFailureRate = ((double)config.failurePercentageEjection.threshold) / 100;
         if (tracker.failureRate() > maxFailureRate) {
+          logger.log(ChannelLogLevel.DEBUG,
+                  "FailurePercentage algorithm detected outlier: {0}", tracker);
```

Comment:
Can we please get the parameters for the failure percentage algorithm logged the same way you did with success rate? After that I think we can get this merged.